src/Okta.Sdk.IntegrationTests/LogsScenarios.cs

```diff
@@ -0,0 +1,114 @@
+﻿using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    [Collection(nameof(ScenariosCollection))]
+    public class LogsScenarios : ScenarioGroup
+    {
+        [Fact]
+        public async Task GetLogs()
+        {
+            var client = GetClient();
+            var createdApp = await client.Applications.CreateApplicationAsync(new CreateBasicAuthApplicationOptions()
+            {
+                Label = "App creation to be logged",
+                Url = "https://example.com/login.html",
+                AuthUrl = "https://example.com/auth.html",
+            });
+
+            try
+            {
+                var logs = await client.Logs.GetLogs().ToList();
+
+                logs.Should().NotBeNullOrEmpty();
+            }
+            finally
+            {
+                await client.Applications.DeactivateApplicationAsync(createdApp.Id);
+                await client.Applications.DeleteApplicationAsync(createdApp.Id);
+            }
+        }
+
+        [Fact]
+        public async Task GetLogsByQueryString()
+        {
+            var client = GetClient();
+            var createdApp = await client.Applications.CreateApplicationAsync(new CreateBasicAuthApplicationOptions()
+            {
+                Label = "App creation to be logged",
+                Url = "https://example.com/login.html",
+                AuthUrl = "https://example.com/auth.html",
+            });
+
+            try
+            {
+                var query = "App creation to be logged";
+                var logs = await client.Logs.GetLogs(null, null, null, query).ToList();
```

Comment:
Does this get put in the `q=` parameter?

Reply:
yes